de-lib/src/main/java/org/iplantc/de/analysis/client/presenter/AnalysesPresenterImpl.java

```diff
@@ -394,8 +394,7 @@ public void onShareSelected(List<Analysis> selected) {
         final AnalysisSharingPresenter sharingPresenter = new AnalysisSharingPresenter(analysisService,
```

Comment:
There's more refactoring that can be done so that we're injecting the sharing presenter, but since the App,Analysis, and Data sharing presenters all share the same interface, I think I'll save those changes for one PR together.